yarn.lock

```diff
@@ -1727,6 +1727,24 @@
     "@types/yargs" "^15.0.0"
     chalk "^4.0.0"
 
+"@kaizen/component-library@^9.6.1":
```

Comment:
Any idea why this appeared?

We probably don't want an outdated component-library version being used inside the monorepo